### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_Keyboard: The interactive elements such as “64 bit” and “32 bit” are not actionable using keyboard.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2798]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated : MAS 2.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate through the "Download Insiders" screen and navigate to “64 bit” and “32 bit” interactive elements.
5.	Observe if these elements are actionable using keyboard.

**Actual:**
When keyboard users navigates to the interactive elements such as “64 bit” and “32 bit” they are not actionable using keyboard instead they are actionable for mouse users.

Similar issue was observed in Edge as well as in Chrome.

**Expected:**
The keyboard users should be able to activate the interactive elements such as “64 bit” and “32 bit” using keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the interactive elements are not actionable using keyboard then the keyboard users might not be able to access the information which is available to the mouse users.

**MAS Reference:**
MAS 2.1.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}) 

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_Keybaord Element not actionable using keyboard..pptx](https://github.com/microsoft/vscode-docs/files/3320402/Code.visualstudio.com_DownloadInsiders_Keybaord.Element.not.actionable.using.keyboard.pptx)
